### PR TITLE
ACMS-000: Removed checklistapi patches.

### PR DESCRIPTION
**Pull request by deepakmishra2**
**Motivation**

Fixes ACMS-000

**Proposed changes**
checklistapi patches used in acquia_cms_common are now merged in the module and hence removing this fixes ACMS installation issue.

### CODE REVIEW

File: modules/acquia_cms_common/composer.json [hunk 1]
```diff
@@ -100,11 +100,6 @@
             },
             "drupal/focal_point": {
                 "3162210 - Preview link accidentally closes the media library": "https://www.drupal.org/files/issues/2020-10-06/3162210-17.patch"
-            },
-            "drupal/checklistapi": {
```
Comment (rajeshreeputra) [inline modules/acquia_cms_common/composer.json:104]: first patch is not available in 2.1.x, so we can not remove it.
cc @chandan-singh7929

File: modules/acquia_cms_common/composer.json [hunk 2]
```diff
@@ -100,11 +100,6 @@
...
-                "2311855: Count the default setting on the total percentage counter": "https://www.drupal.org/files/issues/2021-10-26/checklistapi-count-default-values-forprogress-2311855-2.patch",
```
Comment (rajeshreeputra) [inline modules/acquia_cms_common/composer.json:105]: first patch is not available in 2.1.x, so we can not remove it.
cc @chandan-singh7929

Comment (deepakmishra2) [reply modules/acquia_cms_common/composer.json:105]: yup , only second patch is merged, hence added the first patch back.

**Comment by rajeshreeputra** [comment]
This can be closed as patch is not required, as per latest 2.1.x release of checklistapi.

**Pull request closed without merging**